website/docs/contributing/frontend/ADR/preferred-styling-method.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "ADR: preferred styling method"
+---
+
+This document supersedes ...
+
+<!-- @foseberg: if this supersedes another description, it probably shouldn't be empty, right? -->
```

Comment:
Is this file intentionally left empty? There's a file pointing to it, say that this file supersedes that one, so this one _should_ presumably have some content in it?